pkg/nimble/Makefile.dep

```diff
@@ -42,3 +43,14 @@ ifneq (,$(filter nimble_scanlist,$(USEMODULE)))
   USEMODULE += nimble_addr
   USEMODULE += bluetil_ad
 endif
+
+ifneq (,$(filter nimble_netif,$(USEMODULE)))
+  USEMODULE += l2util
+  USEMODULE += nimble_svc_ipss
+  ifneq (,$(filter gnrc_ipv6_router_default,$(USEMODULE)))
+    USEMODULE += gnrc_sixlowpan_router_default
```

Comment:
I'm sorry, I know I told you to do so, but this pulls in `gnrc_sixlowpan_frag` in as well. Better do it by hand:

```suggestion
  USEMODULE += gnrc_ipv6_nib_6lr
  USEMODULE += gnrc_sixlowpan
  USEMODULE += gnrc_sixlowpan_iphc
```

Reply:
Also considering that that RFC 7668 [explicitly forbids 6LRs](https://tools.ietf.org/html/rfc7668#section-3) maybe make it `gnrc_ipv6_nib_6ln` instead of `gnrc_ipv6_nib_6lr`.

Reply:
yes, its not quite to the RFC to allow this, but the new ipv6mesh-over-ble draft allows this, and I use this for experiments...

Reply:
How about making `6LR`-support optional then? Or is that not tested at all?